scripts/fuzz_opt.py

```diff
@@ -27,6 +27,11 @@
 INPUT_SIZE_LIMIT = 250 * 1024
 
 
+def in_bin(tool):
+  root = os.path.abspath(os.path.dirname(os.path.dirname(__file__)))
+  return os.path.join(root, 'bin', tool)
```

Comment:
If I understand this correctly, the fuzzer must now be run from a directory that is a sibling to the `bin` directory. Would it be better to use the options parsed in scripts/test/shared.py to allow access to `--binaryen-bin`? Granted, not all of those options make sense for a fuzzer, but it would also allow maximum flexibility and reduce duplicated effort. The only trick would be to separate the pass-through arguments from those that should be parsed by shared.py as options for the fuzzer itself. #1830 takes this approach but is slightly hacky.

Reply:
Good idea, fixing.